app/src/main/AndroidManifest.xml

```diff
@@ -91,6 +91,10 @@
             </intent-filter>
         </service>
 
+        <receiver
+            android:exported="true"
+            android:name="androidx.mediarouter.media.MediaTransferReceiver" />
```

Comment:
I've tried to figure out what seamless transfer is about. I am still not really sure. I haven't found any material what seamless transfer means, exactly. However, this is the problematic line. Adding this to the manifest makes the Cast session not start (or at least not notify us/CastSessionManager of the session start). I suspect by adding this we are overriding a behavior which makes the Cast-MediaRoute-related plumbing. 

I have commented on the guidance doc to see if I can get some assistance from the Seoul team on this. I would think they are the ideal people to look into this.

Reply:
After asking around, from Olly who also is in the seamless transfer doc:

> For seamless transfer and the part of the Cast SDK that does routing to work together properly, the Cast SDK needs updating. Before the Cast SDK can be updated, the MediaRouter AndroidX library needs updating. So there is a chain of pending things that need to happen.

So it would seem this patch cannot go in until we update the Cast SDK to the right version?

Reply:
Thanks very much for investigating. That seems to be the case.